services/core-api/app/api/mines/reports/resources/mine_report_definition_resource.py

```diff
@@ -1,6 +1,6 @@
 from flask_restx import Resource, reqparse
 from flask import request
-from werkzeug.exceptions import BadRequest
```

Comment:
As with the `mine_report_definition_list_resource` the changes here are largely git being confused.  All of the removals are the parts that are from the renamed file.  Only the additions here are new.